### ISSUE TITLE
NullReferenceException on delegates

### ISSUE THREAD

**Issue by ste-art** [ArxOne/MrAdvice#54]
```C#
    [MyAdvice]
    class Foo
    {
        private delegate bool Bar();
    }
```
Causes:
```
Error	4	Fody/MrAdvice: Error while weaving method 'System.IAsyncResult Example.Foo/Bar::BeginInvoke(System.AsyncCallback,System.Object)': System.NullReferenceException: Object reference not set to an instance of an object.
   at ArxOne.MrAdvice.Weaver.AspectWeaver.WeaveAdvices(MarkedNode markedMethod, Types types)
   at ArxOne.MrAdvice.Weaver.AspectWeaver.WeaveMethod(ModuleDefinition moduleDefinition, MarkedNode markedMethod, TypeDefinition adviceInterface, Types types)
```
And three more errors.

**Comment by picrap** [comment]
Ouch!

**Comment by picrap** [comment]
Fixed in 1.2.7
